### ISSUE TITLE
reacts when hovering over a grid.

### ISSUE THREAD

**Issue by lhj0621** [nhn/tui.grid#763]
**Describe the bug**

The data in the grid is obtained from api-readData-url, and it is the logic that determines the value of each row when the window is on and puts the value of 'normal' in stts if it is normal. When the above logic is executed, the following phenomenon occurs.

1. Mouse over the grid to display the value
2. When the mouse is moved to a certain position, the background of an unrelated grid field turns red.

>그리드 안의 데이터는 api-readData-url에서 받아오고, 윈도우 on load 시 각 row의 값을 판별하여 정상인 경우 stts에 '정상'이라는 값을 넣는 로직입니다. 위와 같은 로직을 실행했을 때 아래 와 같은 현상이 발생합니다.
>1. 그리드에 마우스를 올려야 값이 뜨는 현상
>2. 특정 위치에 마우스를 올릴 경우 관련 없는 그리드의 필드의 배경이 빨간색이 되는 현상.

**To Reproduce**
```
{
    header: '영문명',
    name: 'ename',
    editor: 'text',
    valign: 'middle',
    editOptions: {
        type: 'text'
    },
    validation: {
        required: true,
        regExp: /[^ㄱ-ㅎ|ㅏ-ㅣ|가-힣]/g
    },
    sortingType: 'desc',
    sortable: true
}
...
{
    header: '상태',
    name: 'stts',
    align: 'center',
    valign: 'middle',
    validation:{
        validatorFn: value => value == '정상'
    },
    width: 250,    
    sortingType: 'desc',
    sortable: true
}
```
**Expected behavior**
![](https://github.com/lhj0621/imagetemp/blob/master/2019-10-23%2013;58;32.gif?raw=true)
**Screenshots**

**Desktop (please complete the following information):**
window 10
chrome
toast 4.6.1

**Comment by js87zz** [comment]
@lhj0621 
We need more detail information to solve this issue.
Add your source related grid not only grid option, but also event handler, sample data and so on.
if it's possible, writing your code in `codepen` or `jsfiddle` is more helpful. :)

**Comment by lhj0621** [comment]
@js87zz 
my code writing in [codepen](https://codepen.io/lhj0621/pen/GRRvdYM?editors=1111).
please help

![](https://github.com/lhj0621/imagetemp/blob/master/19627-gv41a.gif?raw=true)

**Comment by js87zz** [comment]
@lhj0621 
`stts` field is not defined in `gridData` array, for that reason `stts` data couldn't be observable data initially. To solve this issue simply, just define the `stts` value as empty string like below. 
```js
const gridData = [
  {
    name: '이름1',
    ename: 'ename1',
    jumin: '901025-1234562',
    stts: '
  },
  // ...
]
```
However, we need to fix this issue to make the data observable In case of non defining the field in `data` option. Thank you!

**Comment by moonchanyong** [comment]
Hi
would I take this issue?
the bug is reproduced when columns are changed.
as said by @js87zz, added data couldn't be observable data initially. 
so, how about observable be made by name of columns?
and when the column changed, observable has to be reproduced.

**Comment by js87zz** [comment]
@moonchanyong 
You mean raising PR about this issue?

**Comment by moonchanyong** [comment]
> @moonchanyong
> You mean raising PR about this issue? or just talk about this issue?

@js87zz 
I want to raise PR about this issue. can I do it?

**Comment by js87zz** [comment]
@moonchanyong 
Yes! As you mentioned, It could be fixed to make the missed data observable using the column option.
Thank you for your contribution :)

**Comment by js87zz** [comment]
@lhj0621 
The bug has been fixed in `v4.10.1`(#914).
